packages/pwa/dom.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var hopsConfig = require('hops-config');
+
+module.exports = function installServiceWorker() {
+  return new Promise(function(resolve, reject) {
+    if (!('serviceWorker' in navigator)) {
+      return reject(
+        new Error('ServiceWorkers are not supported in this browser')
+      );
+    }
+
+    if (
+      window.location.protocol === 'https' ||
```

Comment:
Should we have Service Workers in development and production builds? Or do we need to treat these differently?

Reply:
We _do_ support SSL in both prod and dev environments, so I would prefer keeping things simple, here.